### PR TITLE
Create docker-connector

### PR DESCRIPTION
**Pull request by linhpd-axonivy**

### CODE REVIEW

**Comment by linhpd-axonivy** [comment]
Hi @weissreto 
Could you please supply the logo and meta.json for the docker-connector?

**Comment by weissreto** [comment]
Hi @linhpd-axonivy 
Done

**Comment by linhpd-axonivy** [comment]
Hi @ivy-sgi , @andreasbalsiger
Please help me review the short description, tags, icon and so on of new docker connector product.

File: market/connector/docker-connector/meta.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
+{
+    "$schema": "https://json-schema.axonivy.com/market/10.0.0/meta.json",
+    "id": "docker-connector",
+    "name": "Docker Connector",
+    "description": "Use the Docker Connector to create, start, stop, and remove Docker containers.",
```
Comment (andreasbalsiger) [inline market/connector/docker-connector/meta.json:5]: I would go for (@linhpd-axonivy): Use this connector to create, start, stop, and remove Docker containers.

**Comment by andreasbalsiger** [comment]
Hi @linhpd-axonivy and @ivy-sgi. I added an inline comment to the description in the meta.json. Rest is great, its the new docker logo and therefore good to go

**Comment by alexsuter** [comment]
@weissreto @linhpd-axonivy 
Hey Guys. You broke this build. You can not merge PR's when the build is yellow. Please fix it.

**Pull request merged**